eos-core-depends

```diff
@@ -31,6 +31,7 @@ bolt
 # Provides the default --init executable for podman
 catatonit
 cdrdao
+console-data
```

Comment:
To be clear, adding `console-data` is not sufficient to get Dvorak (and other non-US English) keyboard layouts working automatically at the VT. And Dvorak (and other non-US English) keyboard layouts work fine in a graphical session. But let's do it.